compiler/damlc/tests/BUILD.bazel

```diff
@@ -1000,3 +1000,21 @@ daml_compile_with_dalf(
     # PlatformIndependence.daml.
     ghc_options = [],
 )
+
+# Test for interfaces + data-dependencies
+daml_build_test(
```

Comment:
I'm not sure if this is the best way to achieve this but I want to have a project which defines an interface and then a separate project which defines a template that implements said interface